### PR TITLE
ICARTT: support non-whole-number data interval, apply scaling factor

### PR DESCRIPTION
**Pull request by ytangnoaa**
Fixed the bug that does not consider the ICARTT's scaling factors and  datainterval in float-point time.

### CODE REVIEW

**Comment by ytangnoaa** [comment]
It is a bug fix to deal with some ICARTT files

File: monetio/profile/icartt.py [hunk 1]
```diff
@@ -436,7 +436,7 @@ def __nan_miss_float(self, raw):
             v = x.replace(self.VAR[i].miss, "NaN")
             if "NaN" in v:
                 v = "NaN"
-            vals.append(float(v.strip()))
+            vals.append(float(v.strip())*self.VAR[i].scale)  # multiply with scaling factor
```
Comment (zmoon) [inline monetio/profile/icartt.py]:
```suggestion
            vals.append(float(v.strip()) * self.VAR[i].scale)  # multiply with scaling factor
```

Comment (zmoon) [reply monetio/profile/icartt.py]: Linting should pass with this change.

**Review by zmoon — APPROVED** [review]
Looks good to me

**Pull request merged**